controller/deis/settings.py

```diff
@@ -136,6 +140,8 @@
     'django.contrib.sites',
     'django.contrib.staticfiles',
     # Third-party apps
+    'django_auth_ldap',
+    'django_fsm',
```

Comment:
We don't use django_fsm any more, so it shouldn't be necessary. The only reason we keep it around any more are for legacy South migrations. Let's remove it so the module's not taking up precious resources when the server boots. django_auth_ldap should be kept though, of course.